src/components/masters/program/main-content/course-cards/BaseCourseCard.jsx

```diff
@@ -65,7 +66,6 @@ class BaseCourseCard extends Component {
     const {
       modals,
       hasEmailsEnabled,
-      hasNewEmailSettings,
```

Comment:
Since we no longer need `hasNewEmailSettings` in state, we should also remove it where we initially set it to `false` at the top of the class.

Reply:
Similarly, since the `BaseCourseCard` no longer shows the inline "success" message after saving email settings, we should remove the `componentDidUpdate` lifecycle method which relates to that message (no longer relevant).